cypress/e2e/home/home-page.spec.cy.ts

```diff
@@ -2,8 +2,15 @@ import PageSteps from '../../steps/page-steps';
 
 describe('Describes "ontotext-yasgui-web-component" component', () => {
 
+   beforeEach(() => {
+      cy.intercept('/repositories/test-repo', {fixture: '/queries/default-query-response.json'}).as('getGuides');
+   });
+
    it('Should load "ontotext-yasgui-web-component" component', () => {
       PageSteps.visitHomePage();
       cy.getByDataSelector('ontotext-yasgui-tag');
+      cy.get('.yasqe_queryButton').click();
```

Comment:
We should move these in the steps to abstract the view access

Reply:
done